chapters/ja/chapter1/3.mdx

```diff
@@ -0,0 +1,323 @@
+
+デフォルトでは、このpipelineは英語の感情分析用にファインチューニングされた特定の事前学習モデルを使用します。このモデルは `classifier` オブジェクトを作成する際にダウンロードされ、キャッシュされます。コマンドを再実行すると、キャッシュされたモデルが代わりに使用され、モデルを再度ダウンロードする必要はありません。
+
+pipelineにテキストを渡す場合、主に3つのステップがあります。
+
+1. テキストはモデルが理解できる形式に前処理される。
+2. 前処理された入力がモデルに渡される。
+3. 予測結果を理解できるように、モデルの後処理が行われる。
+
+現在[利用可能なpipeline](https://huggingface.co/transformers/main_classes/pipelines.html)の一部を紹介します。
+
+- `feature-extraction` (テキストのベクトル表現を取得)
+- `fill-mask`
+- `ner` (固有表現認識)
+- `question-answering`
+- `sentiment-analysis`
+- `summarization`
+- `text-generation`
+- `translation`
+- `zero-shot-classification`
+
+では、いくつか見ていきましょう！
+
+## ゼロショット分類
+
+まず、ラベル付けされていないテキストを分類する必要があるような、より困難なタスクに取り組むことから始めます。これは実際のプロジェクトでよくあるシナリオです。なぜなら、テキストにアノテーションをつけるのは通常時間がかかり、専門知識が必要だからです。このような場合、`zero-shot-classification` pipelineは非常に強力です。分類に使用するラベルを指定できるので、事前に学習したモデルのラベルに依存する必要がありません。肯定的か否定的かの2つのラベルを使って、モデルがどのようにテキストを分類するかは既に見たとおりです。しかし、他の任意のラベルセットを使ってテキストを分類することもできます。
+
+
+```python
+from transformers import pipeline
+classifier = pipeline("zero-shot-classification")
+classifier(
+    "This is a course about the Transformers library",
+    candidate_labels=["education", "politics", "business"],
+)
+```
+
+```python out
+{'sequence': 'This is a course about the Transformers library',
+ 'labels': ['education', 'business', 'politics'],
+ 'scores': [0.8445963859558105, 0.111976258456707, 0.043427448719739914]}
+```
+
+このpipelineは _zero-shot_ と呼ばれます。なぜなら、これを使うために自前のデータセットでモデルのファインチューニングをする必要がないからです。任意のラベルのリストに対して直接確率スコアを返すことができます。
+
+<Tip>
+
+✏️ **試してみよう!** 独自の入力とラベルで遊んでみて、モデルがどのように振る舞うか見てみましょう。
+
+</Tip>
+
+## 文章生成
+
+では、pipelineを使ってテキストを生成する方法を見てみましょう。主なアイデアは、プロンプトを与えると、モデルが残りのテキストを生成してそれを補完することです。これは、多くの携帯電話に搭載されている予測入力機能に類似しています。テキスト生成にはランダム性が含まれるため、以下と同様な結果が得られないのが通常です。
+
+```python
+from transformers import pipeline
+generator = pipeline("text-generation")
+generator("In this course, we will teach you how to")
+```
+
+```python out
+[{'generated_text': 'In this course, we will teach you how to understand and use '
+                    'data flow and data interchange when handling user data. We '
+                    'will be working with one or more of the most commonly used '
+                    'data flows — data flows of various types, as seen by the '
+                    'HTTP'}]
+```
+
+引数`num_return_sequences`で異なるシーケンスの生成数を、引数`max_length`で出力テキストの合計の長さを制御することができます。
+
+<Tip>
+
+
+✏️ **試してみよう!** `num_return_sequences` と `max_length` 引数を用いて、15語ずつの2つの文を生成してみましょう！
+
+</Tip>
+
+
+## pipelineでHubから任意のモデルを使用する
+
+これまでの例では、タスクに応じたデフォルトのモデルを使用しましたが、特定のタスク（例えばテキスト生成）のpipelineで使用するモデルをHubから選択することも可能です。[Model Hub](https://huggingface.co/models)にアクセスし、左側の対応するタグをクリックすると、そのタスクでサポートされているモデルのみが表示されます。[このようなページ](https://huggingface.co/models?pipeline_tag=text-generation)が表示されるはずです。
+
+それでは、[`distilgpt2`](https://huggingface.co/distilgpt2)モデルを試してみましょう! 先ほどと同じpipelineでロードする方法を説明します。
+
+```python
+from transformers import pipeline
+generator = pipeline("text-generation", model="distilgpt2")
```

Comment:
maybe we could use one model from https://huggingface.co/models?language=ja&pipeline_tag=text-generation&sort=downloads ?

Reply:
I think it's best to leave the source files with English models / outputs and translate the notebooks themselves with Japanese models (if desired).

This has the advantage of ensuring parity between the English <--> Japanese translations and means you don't have to reword a lot of text to explain e.g. the inputs / outputs.

See [here](https://github.com/huggingface/notebooks/pull/247) for an example of how this was done for the French translation (but it's certainly not a requirement for this great translation effort!)

Reply:
Yes it makes sense!